cli/client.js

```diff
@@ -11,13 +11,15 @@ const argv = mri(process.argv.slice(2), {
 if (argv.help || argv.h) {
 	process.stdout.write(`
 Usage:
-    tcp-over-websockets <tunnel-url> <tunnelled-target> <port-to-listen-on>
+    tcp-over-websockets <tunnel-url> <tunnelled-target> <port-to-listen-on> [<http-header-for-tunnel>]
```

Comment:
I prefer specifying the headers as optional `--header`/`-H` flags, just [like curl does it](https://stackoverflow.com/a/4212535/1072129).

```suggestion
    tcp-over-websockets [options] <tunnel-url> <tunnelled-target> <port-to-listen-on>
```

Then, add a new section "Options" after "Arguments" that explains what `--header`/`-H` does.

`mri`, the args parsing library used above, will parse `-H foo -H bar` just fine as `{headers: ['foo', 'bar']}` (except that it [has a bug when the alias and the long-form are mixed](https://github.com/lukeed/mri/issues/23)).

Reply:
done